### DISCUSSION TITLE
Bungeecord online mode

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#662] [Help] [answered]
![image](https://user-images.githubusercontent.com/71212639/153027592-50b434c3-5075-4bc1-ba75-3c2283d02fd8.png)
How can I set the server to online mode? (I have set the bungeecord to online mode)

**Comment by MrlnHi** [comment]
The error message suggests that your server is currently in online mode, which you do not want. Do you call MojangAuth.init() in your code somewhere? If that is the case, remove that line and try again. Also use velocity if you can

**Reply by DevBlokJochem** [reply]
yes, but I have turned the bungeecord also to online mode.

**Reply by MrlnHi** [reply]
That should not matter. Do you call MojangAuth.init() in your minestom code?

**Reply by DevBlokJochem** [reply]
Yes

**Reply by DevBlokJochem** [reply]
The reason why Im not using velocity is because I want a simple & quick setup  to test how my game is playing in 1.8/1.12

**Comment by emortaldev** [comment, marked as answer]
(afaik) BungeeCord requires you to have backend servers as offline mode (MojangAuth is essentially online-mode), as Bungee does the mojang stuff
